### PR TITLE
Feature/core 10913 fix radio button geo tree item

### PR DESCRIPTION
**Pull request by irmakyz**

### CODE REVIEW

File: src/elements/GeoTree/GeoTreeItem.vue [hunk 1]
```diff
@@ -42,7 +42,15 @@
           />
         </span>
         <input
-          v-if="!isItemInputHidden"
+          v-if="isSingleSelectMode && isSingleItem"
```
Comment (Navas23) [inline src/elements/GeoTree/GeoTreeItem.vue:45]: Why do we need to have 2 inputs?

File: src/elements/GeoTree/GeoTree.examples.md [hunk 2]
```diff
@@ -125,6 +125,137 @@ export default {
 </script>
 ```
 
+### Basic example with single select
+
+```vue live
+<template>
```
Comment (Navas23) [inline src/elements/GeoTree/GeoTree.examples.md:131]: Why do we need another single select example and not use the current one?

**Review by Navas23 — COMMENTED** [review]
I don't understand the solution, why not stopping the propagation is the solution?

File: src/elements/GeoTree/GeoTreeItem.vue (hunk 1 above)
Comment (irmakyz) [reply src/elements/GeoTree/GeoTreeItem.vue:45]: hey, because i couldn't find a way to set @click.stop as a conditional way,  so did as this way

File: src/elements/GeoTree/GeoTree.examples.md (hunk 2 above)
Comment (irmakyz) [reply src/elements/GeoTree/GeoTree.examples.md:131]: all examples are based on checkboxes, so i added one with radio input

**Comment by irmakyz** [comment]
> I don't understand the solution, why not stopping the propagation is the solution?

i searched on the internet and it says when we stop the event propagation, it effects the functionality of the radio inputs, that's why i removed it and it works now

File: src/elements/GeoTree/GeoTree.examples.md (hunk 2 above)
Comment (Navas23) [reply src/elements/GeoTree/GeoTree.examples.md:131]: There is one with radio inputs

File: src/elements/GeoTree/GeoTreeItem.vue (hunk 1 above)
Comment (Navas23) [reply src/elements/GeoTree/GeoTreeItem.vue:45]: But what is happening if we keep the click.stop? the event is not being sent? have you checked the input event instead of the click one?

File: src/elements/GeoTree/GeoTree.examples.md (hunk 2 above)
Comment (irmakyz) [reply src/elements/GeoTree/GeoTree.examples.md:131]: my bad deleting this one

File: src/elements/GeoTree/GeoTree.examples.md [hunk 3]
```diff
@@ -800,134 +800,4 @@ export default {
   }
 }
 </script>
-```
-
-### With max number of items checked
```
Comment (Navas23) [inline src/elements/GeoTree/GeoTree.examples.md:805]: Why did you remove this one?

File: src/elements/GeoTree/GeoTreeItem.vue (hunk 1 above)
Comment (irmakyz) [reply src/elements/GeoTree/GeoTreeItem.vue:45]: yes i checked both of them, the event is not being sent

Comment (Navas23) [reply src/elements/GeoTree/GeoTreeItem.vue:45]: And why don't we capture the click normally and if it's singleSelectMode we stop the propagation?

Comment (irmakyz) [reply src/elements/GeoTree/GeoTreeItem.vue:45]: hmm, yeah is a better idea maybe, i will change

Comment (irmakyz) [reply src/elements/GeoTree/GeoTreeItem.vue:45]: - [ ] capture the click normally and if it's not singleSelectMode we stop the propagation

File: src/elements/GeoTree/GeoTreeItem.vue [hunk 4]
```diff
@@ -42,13 +42,13 @@
           />
         </span>
         <input
-          v-if="!isItemInputHidden"
+          v-if="!isInputDisabled"
```
Comment (Navas23) [inline src/elements/GeoTree/GeoTreeItem.vue:45]: Why this change? this will break stuff for sure

Comment (irmakyz) [reply src/elements/GeoTree/GeoTreeItem.vue]: - [ ] fix wrong change

**Review by Navas23 — APPROVED** [review]

**Review by dletran — APPROVED** [review]
